lib/rubocop/cop/layout/space_around_equals_in_parameter_default.rb

```diff
@@ -70,7 +70,7 @@ def no_surrounding_space?(arg, equals)
           !arg.space_after? && !equals.space_after?
         end
 
-        def message(_)
+        def message(_msg)
```

Comment:
The argument to `#message` is not a message. It is the offending node. 🙂

Reply:
Thanks! I'll change it here and everywhere else this PR corrects a `#message`.